### PR TITLE
Basics for sms transport

### PR DESCRIPTION
**Pull request by justinvdm**

### CODE REVIEW

**Comment by justinvdm** [comment]
ready for review @codiebeulaine && (@hodgestar || @rudi)

File: vxvas2nets/vas2nets_sms.py [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+
+        # TODO publish status
+
+    @inlineCallbacks
+    def handle_inbound_message(self, message_id, request, vals):
+        yield self.publish_message(
+            **self.get_message_dict(message_id, vals))
+
+        self.respond(message_id, http.OK, {})
+
```
Comment (hodgestar) [inline vxvas2nets/vas2nets_sms.py:159]: Do we need a TODO for publishing a successful status here?

Comment (justinvdm) [inline vxvas2nets/vas2nets_sms.py:159]: Ah, thanks, will add

File: vxvas2nets/tests/test_vas2nets_sms.py [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+            }
+        })
+
+    @inlineCallbacks
+    def test_inbound_decode_error(self):
+        with LogCatcher() as lc:
+            res = yield self.tx_helper.mk_request(
+                sender='+123',
+                receiver='456',
+                msgdata=u'ポケモン'.encode('utf-16'),
```
Comment (hodgestar) [inline vxvas2nets/tests/test_vas2nets_sms.py:111]: Pikathulhu

Comment (justinvdm) [inline vxvas2nets/tests/test_vas2nets_sms.py:111]: :D

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**